robot/board.py

```diff
@@ -168,4 +168,10 @@ def close(self):
         """
         self.socket.detach()
 
+    def __str__(self):
+        return "{} - {}".format(
+            self.__name__,
+            self.serial
```

Comment:
Missing trailing comma here (though I actually think this would fit on a single line?)